docs/android/emulator/device-manager.md

```diff
@@ -0,0 +1,426 @@
+
+:::image type="content" source="media/device-manager/win/01-devices-dialog-sml.png" alt-text="Screenshot of the Android Device Manager in the Devices tab." lightbox="media/device-manager/win/01-devices-dialog.png":::
+
+## Requirements
+
+To use the Android Device Manager, you'll need the following items:
+
+- Visual Studio 2022 version 17.1 Preview 3. Visual Studio Community, Professional, and Enterprise editions are supported.
+
+- The **Android SDK API Level 30** or later. Be sure to install the Android SDK at its default location if it isn't already installed: _C:\\Program Files (x86)\\Android\\android-sdk_.
+
+- The following packages must be installed in the:
+
+  - **Android SDK Tools 5.0** or later
+  - **Android SDK Platform-Tools 31.0.3** or later
+  - **Android SDK Build-Tools 30.0.2** or later
+  - **Android Emulator 30.8.4** or later
+
+  These packages should be displayed with **Installed** status as seen in the following screenshot:
+
+  :::image type="content" source="media/device-manager/win/02-sdk-tools.png" alt-text="Installing Android SDK Tools.":::
+
+When you install the **Mobile development with .NET** workload in Visual Studio, and select the optional **.NET MAUI (Preview)** workload, everything is installed for you. For more information on setting up .NET MAUI with Visual Studio, see [Installation](../../get-started/installation.md).
+
+## Open the device manager
+
+Open the Android Device Manager in Visual Studio from the **Tools** menu by pressing **Tools > Android > Android Device Manager**:
+
+:::image type="content" source="media/device-manager/win/03-tools-menu.png" alt-text="Launching the Device manager from the Tools menu.":::
+
+## Main screen
+
+When you run the Android Device Manager, it presents a screen that displays all currently-configured virtual devices. For each virtual device, the **Name**, **OS** (Android Version), **Processor**, **Memory** size, and screen **Resolution** are displayed:
+
+When you select a device in the list, the **Start** button appears on the right. Press the **Start** button to launch the emulator with this virtual device. If the emulator is running with the selected virtual device, the **Start** button changes to a **Stop** button that you can use to halt the emulator.
+
+### Create a new device
+
+To create a new device, press the **New** button:
+
+:::image type="content" source="media/device-manager/win/08-new-button.png" alt-text="New button for creating a new device.":::
+
+The **New Device** window is displayed. To configure the device, follow these steps:
+
+01. Give the device a new name. In the following example, the new device is named _Pixel 2 - API 28_.
+
+    :::image type="content" source="media/device-manager/win/09-device-props-sml.png" alt-text="New Device screen of the Device Manager." lightbox="media/device-manager/win/09-device-props.png":::
```

Comment:
It might be better to go with a Processor field of x86_64. Google Play Store has required x86_64 apps for a while now.